lutris doesn't build

**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior:
1. cd ~/nixpkgs
2. nix-build -A lutris

<details>
  <summary>log</summary>

```
these derivations will be built:
  /nix/store/7k5gzi7arhwz9k8g9022bbpaspz7f7d6-steam-original-1.0.0.61.drv
  /nix/store/5wfqrwb032ihdsr1d7ssxvm9drmva0bd-steam-runtime.drv
  /nix/store/pkadgywwp969pd9xp5lsp6qgvpn6g0v6-steam-runtime-wrapped.drv
  /nix/store/6bpffgjj6v73vn1dy58pr5gss716rhsc-steam-usr-multi.drv
  /nix/store/ra96grssihnlmm6hck37igl7a734lff9-setup.sh.drv
  /nix/store/qk69kbjkygyy5x66kg8jb4sq0j0ljmcd-steam-runtime.drv
  /nix/store/impx98icpsm5p3a2bz9902kmqipqrldz-steam-runtime-wrapped.drv
  /nix/store/yswgc3yp50fzsjqkhafgfn79gajf3i0k-steam-chrootenv-etc.drv
  /nix/store/yi2djl4aanl2p5klp2083hy9fp493r21-steam-usr-target.drv
  /nix/store/2ag8hs41za58gplfs1043wgqb21drl5h-steam-fhs.drv
  /nix/store/rv3sq6b344w5n1mz1qmd5zfj3nwz0z31-steam-wrapper.sh.drv
  /nix/store/lvqsjrsk5y4ki3az11yaiv95sn2s8n0s-steam-init.drv
  /nix/store/ims6j96x5nmin83cqby02rcw263qprnm-steam.drv
  /nix/store/6jkxqxsa0qy5gbd3gyspwwcbn3n1qd5j-steam.drv
  /nix/store/kxq5alc6vq56q2g2nlwyl1qiisfph2n1-ffmpeg-4.3.1.drv
  /nix/store/8qw0ng7dif8gjpykqnwk59kq1pkfc4ap-gst-libav-1.16.2.drv
  /nix/store/x5b84m5q7d969waxqb23p21736zb2ff4-chromaprint-1.5.0.drv
  /nix/store/wridld74drhh1bp58lp0jzacv8iv68pz-gst-plugins-bad-1.16.2.drv
  /nix/store/d4l7iifhbfj8x0rvs4igm5bsx08z4vqk-webkitgtk-2.28.4.drv
  /nix/store/anzysx931vhk226vd4ag3hpidhj4k3cl-lutris-original-0.5.7.1.drv
  /nix/store/i05gfpjdgk59n9g86c59322jc5r9nwa9-qtbase-everywhere-src-5.15.0.tar.xz.drv
  /nix/store/q3myxzliby0m27d4vj5pffszpj2f4816-qtbase-5.15.0.drv
  /nix/store/i22a3r1gycmiamv16baiiz3q4rvm4jvv-hook.drv
  /nix/store/q12g1vlz9al4a02lsl3mzqd89xnxbf6p-qtdeclarative-everywhere-src-5.15.0.tar.xz.drv
  /nix/store/a1q1s4bhdgfmspjvsd0aw2rqj4ya3kq6-qtsvg-everywhere-src-5.15.0.tar.xz.drv
  /nix/store/qkfq3d0k9dhv3jri0d4xwgh9iavzdpd3-qtsvg-5.15.0.drv
  /nix/store/d01k9kmy8s4vv4m29f4b6qv3hrpr5yg2-qtdeclarative-5.15.0.drv
  /nix/store/hfk5nxi7j6lvlylh4c8q9jq3mf66vv6x-qtmultimedia-everywhere-src-5.15.0.tar.xz.drv
  /nix/store/c3fnkq7mxvf6r3jm2y2dj4qnyy6gji45-qtmultimedia-5.15.0.drv
  /nix/store/hf7q7rq1ndrfd9nlbg1qyidf4xr4jxhn-allegro-4.4.3.1.drv
  /nix/store/m9x320ffimx5g60sk26dhid7fhlykds2-ffmpeg-3.4.8.drv
  /nix/store/p6plnvmpzj4zpj5a888n36gf7gzhsg6g-dumb-2.0.3.drv
  /nix/store/h1gw5j09cwshn1lp3387ycaqhyw4lr16-lutris-usr-target.drv
  /nix/store/cjy3h02c4mf7ah7mbff9d2vakikanj9n-lutris-fhs.drv
  /nix/store/1bazi1kvs7n7rag87gf5m4aibanlz4fw-lutris.drv
  /nix/store/dqc6vkzsh0f87zkkx760cwha72yqnmpb-lutris.drv
building '/nix/store/hf7q7rq1ndrfd9nlbg1qyidf4xr4jxhn-allegro-4.4.3.1.drv'...
building '/nix/store/m9x320ffimx5g60sk26dhid7fhlykds2-ffmpeg-3.4.8.drv'...
building '/nix/store/ra96grssihnlmm6hck37igl7a734lff9-setup.sh.drv'...
building '/nix/store/7k5gzi7arhwz9k8g9022bbpaspz7f7d6-steam-original-1.0.0.61.drv'...
unpacking sources
unpacking source archive /nix/store/qcrxlr9sh1crkywwn8rh4ma415k6cf0g-steam_1.0.0.61.tar.gz
building '/nix/store/kxq5alc6vq56q2g2nlwyl1qiisfph2n1-ffmpeg-4.3.1.drv'...
source root is steam
setting SOURCE_DATE_EPOCH to timestamp 1554914951 of file steam/bootstraplinux_ubuntu12_32.tar.xz
patching sources
configuring
no configure script, doing nothing
building
build flags: SHELL=/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/bash DESTDIR=\$\(out\) PREFIX=
Run steam with "./steam" or install it with "sudo make install"
installing
install flags: SHELL=/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/bash DESTDIR=\$\(out\) PREFIX= install
unpacking sources
unpacking source archive /nix/store/g1ikx6gfk3qf5fmgj593rvwblzmn3frj-allegro-4.4.3.1.tar.gz
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/bin/
install -p -m 755 steam /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/bin/
install -p -m 755 steamdeps /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/bin/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/doc/steam/
install -p -m 644 README steam_subscriber_agreement.txt /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/doc/steam/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/man/man6/
install -m 644 steam.6 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/man/man6/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/16x16/apps/
install -p -m 644 icons/16/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/16x16/apps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/24x24/apps/
install -p -m 644 icons/24/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/24x24/apps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/256x256/apps/
install -p -m 644 icons/256/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/256x256/apps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/32x32/apps/
install -p -m 644 icons/32/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/32x32/apps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/48x48/apps/
install -p -m 644 icons/48/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/icons/hicolor/48x48/apps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/pixmaps/
install -p -m 644 icons/48/steam.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/pixmaps/
install -p -m 644 icons/48/steam_tray_mono.png /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/pixmaps/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/lib/steam/
install -p -m 644 bootstraplinux_ubuntu12_32.tar.xz /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/lib/steam/
install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/applications/
install -p -m 644 steam.desktop /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/applications/
if [ -d /etc/apt ]; then \
        install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/etc/apt/sources.list.d/; \
        install -p -m 644 steam.list /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/etc/apt/sources.list.d/; \
        install -d -m 700 /build/steam/gpg; \
        gpg --homedir=/build/steam/gpg --no-default-keyring --keyring=/build/steam/steam.gpg --import steam-key.asc; \
        install -d -m 755 /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/etc/apt/trusted.gpg.d/; \
        install -p -m 644 steam.gpg /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/etc/apt/trusted.gpg.d/; \
        rm -rf /build/steam/steam.gpg* /build/steam/gpg; \
fi
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61
gzipping man pages under /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/share/man/
strip is /nix/store/iygzjy14i075hkxr2gq8cpfjvk2rkx0b-binutils-2.31.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/lib  /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/bin
source root is allegro-4.4.3.1
patching script interpreter paths in /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61
setting SOURCE_DATE_EPOCH to timestamp 1551663043 of file allegro-4.4.3.1/language.dat
/nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61/bin/steam: interpreter directive changed from "/usr/bin/env bash" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/bash"
patching sources
applying patch /nix/store/4k65bb1kygjmddbvl0rwgqcff1hzwrxm-nix-unstable-sandbox-fix.patch
patching file docs/CMakeLists.txt
configuring
fixing cmake files...
checking for references to /build/ in /nix/store/qh8hq1h5rj5iy02pwhpp1xsdgpr7b556-steam-original-1.0.0.61...
building '/nix/store/i05gfpjdgk59n9g86c59322jc5r9nwa9-qtbase-everywhere-src-5.15.0.tar.xz.drv'...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/share/doc/ALLEGRO -DCMAKE_INSTALL_INFODIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/include -DCMAKE_INSTALL_SBINDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_OSX_ARCHITECTURES=x86_64 -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/iygzjy14i075hkxr2gq8cpfjvk2rkx0b-binutils-2.31.1/bin/strip -DCMAKE_RANLIB=/nix/store/iygzjy14i075hkxr2gq8cpfjvk2rkx0b-binutils-2.31.1/bin/ranlib -DCMAKE_AR=/nix/store/iygzjy14i075hkxr2gq8cpfjvk2rkx0b-binutils-2.31.1/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/9jcrjy9qihgmhzzkr1lx3kqbibf9cy7y-allegro-4.4.3.1 -DCMAKE_SKIP_RPATH=ON

trying https://download.qt.io/official_releases/qt/5.15/5.15.0/submodules/qtbase-everywhere-src-5.15.0.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
unpacking sources
unpacking source archive /nix/store/2f4i6697g8vmy4cd875ikdj1mr6zhdd6-ffmpeg-3.4.8.tar.bz2
-- The C compiler identification is GNU 9.3.0
unpacking sources
unpacking source archive /nix/store/2853lgnwdg6dandsckh3rrfz88nh6qi2-ffmpeg-4.3.1.tar.bz2
-- The CXX compiler identification is GNU 9.3.0
-- Detecting C compiler ABI info
100   297  100   297    0     0    617      0 --:--:-- --:--:-- --:--:--   617
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/mimzy2k99xq842k59zl3f2z2s34rcq5b-gcc-wrapper-9.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/mimzy2k99xq842k59zl3f2z2s34rcq5b-gcc-wrapper-9.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE)
-- Check if the system is big endian
-- Searching 16 bit integer
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Searching 16 bit integer - Using unsigned short
-- Check if the system is big endian - little endian
-- Looking for include file dirent.h
-- Looking for include file dirent.h - found
-- Looking for include file inttypes.h
-- Looking for include file inttypes.h - found
-- Looking for include file linux/awe_voice.h
-- Looking for include file linux/awe_voice.h - not found
-- Looking for include file linux/input.h
-- Looking for include file linux/input.h - found
-- Looking for include files sys/types.h, linux/joystick.h
-- Looking for include files sys/types.h, linux/joystick.h - found
-- Looking for include file linux/soundcard.h
-- Looking for include file linux/soundcard.h - found
-- Looking for include file machine/soundcard.h
-- Looking for include file machine/soundcard.h - not found
-- Looking for include file soundcard.h
-- Looking for include file soundcard.h - not found
-- Looking for include file stdint.h
-- Looking for include file stdint.h - found
-- Looking for include file sys/io.h
source root is ffmpeg-3.4.8
-- Looking for include file sys/io.h - found
-- Looking for include file sys/stat.h
setting SOURCE_DATE_EPOCH to timestamp 1593816884 of file ffmpeg-3.4.8/libswscale/yuv2rgb.c
patching sources
patching script interpreter paths in .
./ffbuild/pkgconfig_generate.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./ffbuild/version.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/ffserver-regression.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for include file sys/stat.h - found
-- Looking for include file sys/time.h
./tests/fate.sh: interpreter directive changed from " /bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/regression-funcs.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/lavf-regression.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/fate/source-check.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for include file sys/time.h - found
-- Looking for include file sys/soundcard.h
./tests/fate-run.sh: interpreter directive changed from " /bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/copycooker.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./doc/doxy-wrapper.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./compat/windows/makedef: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./compat/windows/mslink: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./compat/cuda/ptx2c.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for include file sys/soundcard.h - found
-- Looking for include file sys/utsname.h
./compat/solaris/make_sunver.pl: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tools/unwrap-diff: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/gen-rc: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
source root is ffmpeg-4.3.1
./tools/make_chlayout_test: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
-- Looking for include file sys/utsname.h - found
-- Looking for getexecname
./tools/dvd2concat: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
setting SOURCE_DATE_EPOCH to timestamp 1594463972 of file ffmpeg-4.3.1/libavutil/common.h
patching sources
patching script interpreter paths in .
./ffbuild/pkgconfig_generate.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/missing_codec_desc: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/clean-diff: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./ffbuild/version.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/fate.sh: interpreter directive changed from " /bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/patcheck: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tests/fate/source-check.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for getexecname - not found
-- Looking for memcmp
./tests/fate-run.sh: interpreter directive changed from " /bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/plotframes: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tests/copycooker.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/bisect-create: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./doc/doxy-wrapper.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/libav-merge-next-commit: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./compat/windows/makedef: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/murge: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./compat/windows/mslink: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./configure: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
configuring
-- Looking for memcmp - found
-- Looking for mkstemp
./compat/cuda/ptx2c.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
configure flags: --disable-static --prefix=/nix/store/9hllbym74a3wg75qlmwg8pybwbjxlqz1-ffmpeg-3.4.8 --arch=x86_64 --target_os=linux --enable-gpl --enable-version3 --enable-shared --enable-pic --enable-runtime-cpudetect --enable-hardcoded-tables --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --disable-ffplay --enable-ffprobe --disable-ffserver --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avresample --enable-avutil --enable-postproc --enable-swresample --enable-swscale --disable-doc --enable-libass --enable-bzlib --enable-gnutls --enable-fontconfig --enable-libfreetype --enable-libmp3lame --enable-iconv --enable-libtheora --enable-libssh --enable-vaapi --enable-libdrm --enable-vdpau --enable-libvorbis --enable-libvpx --enable-lzma --enable-opengl --enable-libpulse --enable-sdl2 --enable-libsoxr --enable-libx264 --enable-libxvid --enable-zlib --enable-libopus --enable-libspeex --enable-libx265 --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
./compat/solaris/make_sunver.pl: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tools/target_dec_fate.sh: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/unwrap-diff: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/gen-rc: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/cl2c: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for mkstemp - found
-- Looking for mmap
./tools/make_chlayout_test: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tools/dvd2concat: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tools/missing_codec_desc: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/clean-diff: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for mmap - found
-- Looking for mprotect
./tools/patcheck: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/plotframes: interpreter directive changed from "/usr/bin/env perl" to "/nix/store/5b6853g3y2wvzc48l6921a6qcr55lqk2-perl-5.32.0/bin/perl"
./tools/bisect-create: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/libav-merge-next-commit: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
./tools/murge: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
-- Looking for mprotect - found
-- Looking for sched_yield
./configure: interpreter directive changed from "/bin/sh" to "/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/sh"
configuring
configure flags: --disable-static --prefix=/nix/store/1b9v2qg7a19hrvz00kgsir7hqg397s4k-ffmpeg-4.3.1 --arch=x86_64 --target_os=linux --enable-gpl --enable-version3 --enable-shared --enable-pic --enable-runtime-cpudetect --enable-hardcoded-tables --enable-pthreads --disable-w32threads --disable-os2threads --enable-network --enable-pixelutils --enable-ffmpeg --disable-ffplay --enable-ffprobe --enable-avcodec --enable-avdevice --enable-avfilter --enable-avformat --enable-avresample --enable-avutil --enable-postproc --enable-swresample --enable-swscale --disable-doc --enable-libass --enable-bzlib --enable-gnutls --enable-fontconfig --enable-libfreetype --enable-libmp3lame --enable-iconv --enable-libtheora --enable-libssh --enable-vaapi --enable-libdrm --enable-vdpau --enable-libvorbis --enable-libvpx --enable-lzma --enable-opengl --enable-libmfx --disable-libaom --enable-libpulse --enable-sdl2 --enable-libsoxr --enable-libx264 --enable-libxvid --enable-zlib --enable-libopus --enable-libspeex --enable-libx265 --enable-libdav1d --disable-debug --enable-optimizations --disable-extra-warnings --disable-stripping
-- Looking for sched_yield - found
-- Looking for stricmp
-- Looking for stricmp - not found
-- Looking for strlwr
-- Looking for strlwr - not found
-- Looking for strupr
-- Looking for strupr - not found
-- Looking for sysconf
-- Looking for sysconf - found
-- Performing Test ALLEGRO_HAVE_SV_PROCFS_H
-- Performing Test ALLEGRO_HAVE_SV_PROCFS_H - Failed
-- Performing Test ALLEGRO_HAVE_PROCFS_ARGCV
-- Performing Test ALLEGRO_HAVE_PROCFS_ARGCV - Failed
-- Performing Test MAP_FAILED_DEFINED
-- Performing Test MAP_FAILED_DEFINED - Success
-- Performing Test ALLEGRO_USE_CONSTRUCTOR
-- Performing Test ALLEGRO_USE_CONSTRUCTOR - Success
-- Performing Test ALLEGRO_HAVE_POSIX_MONOTONIC_CLOCK
-- Performing Test ALLEGRO_HAVE_POSIX_MONOTONIC_CLOCK - Success
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Failed
-- Looking for pthread_create in pthreads
-- Looking for pthread_create in pthreads - not found
-- Looking for pthread_create in pthread
-- Looking for pthread_create in pthread - found
-- Found Threads: TRUE
-- Performing Test OSS_COMPILES
-- Performing Test OSS_COMPILES - Success
-- Found OSS: 1
-- Looking for alOpenPort in audio
-- Looking for alOpenPort in audio - not found
-- Found X11: /nix/store/620j8vsvyq976lcxxszi99prcg4hh3s5-xorgproto-2019.1/include
-- Looking for XOpenDisplay in /nix/store/liap55k4abd2sw7jfrnaigg1f2wphd03-libX11-1.6.8/lib/libX11.so;/nix/store/79hjq0j4bmxi3zpl1nhfafp0zpdzfpd1-libXext-1.3.4/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/liap55k4abd2sw7jfrnaigg1f2wphd03-libX11-1.6.8/lib/libX11.so;/nix/store/79hjq0j4bmxi3zpl1nhfafp0zpdzfpd1-libXext-1.3.4/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
-- Looking for XOpenIM in X11
-- Looking for XOpenIM in X11 - found
-- Looking for XDGAQueryExtension in Xxf86dga
-- Looking for XDGAQueryExtension in Xxf86dga - found
CMake Warning (dev) at /nix/store/6hicvfkq4v2gkbiv1rc2lp1vd8ji4x8l-cmake-3.18.2/share/cmake-3.18/Modules/FindOpenGL.cmake:305 (message):
  Policy CMP0072 is not set: FindOpenGL prefers GLVND by default when
  available.  Run "cmake --help-policy CMP0072" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.

  FindOpenGL found both a legacy GL library:

    OPENGL_gl_LIBRARY: /nix/store/acc72d5a2jsalp9cxl0jw22mz4qgyc6f-libGL-1.3.2/lib/libGL.so

  and GLVND libraries for OpenGL and GLX:

    OPENGL_opengl_LIBRARY: /nix/store/acc72d5a2jsalp9cxl0jw22mz4qgyc6f-libGL-1.3.2/lib/libOpenGL.so
    OPENGL_glx_LIBRARY: /nix/store/acc72d5a2jsalp9cxl0jw22mz4qgyc6f-libGL-1.3.2/lib/libGLX.so

  OpenGL_GL_PREFERENCE has not been set to "GLVND" or "LEGACY", so for
  compatibility with CMake 3.10 and below the legacy GL library will be used.
Call Stack (most recent call first):
  addons/allegrogl/CMakeLists.txt:6 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found OpenGL: /nix/store/acc72d5a2jsalp9cxl0jw22mz4qgyc6f-libGL-1.3.2/lib/libOpenGL.so
-- Looking for glXGetProcAddress
-- Looking for glXGetProcAddress - found
-- Found ZLIB: /nix/store/2gci5q5yz717xycp444wyjxki4mbwwav-zlib-1.2.11/lib/libz.so (found version "1.2.11")
-- Found PNG: /nix/store/dwy1f99xzdqgzxx7b96xw4fjdmhml7vf-libpng-apng-1.6.37/lib/libpng.so (found version "1.6.37")
CMake Warning (dev) at /nix/store/6hicvfkq4v2gkbiv1rc2lp1vd8ji4x8l-cmake-3.18.2/share/cmake-3.18/Modules/FindPackageHandleStandardArgs.cmake:273 (message):
  The package name passed to `find_package_handle_standard_args` (VORBIS)
  does not match the name of the calling package (Vorbis).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindVorbis.cmake:21 (find_package_handle_standard_args)
  addons/logg/CMakeLists.txt:6 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found VORBIS: /nix/store/jmchnw30aij8c2jdxjqf6whhy428mbji-libogg-1.3.4-dev/include
-- Configuring done
-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_INSTALL_BINDIR
    CMAKE_INSTALL_DOCDIR
    CMAKE_INSTALL_INCLUDEDIR
    CMAKE_INSTALL_INFODIR
    CMAKE_INSTALL_LIBDIR
    CMAKE_INSTALL_LIBEXECDIR
    CMAKE_INSTALL_LOCALEDIR
    CMAKE_INSTALL_MANDIR
    CMAKE_INSTALL_OLDINCLUDEDIR
    CMAKE_INSTALL_SBINDIR


-- Build files have been written to: /build/allegro-4.4.3.1/build
cmake: enabled parallel building
building
build flags: -j4 -l4 SHELL=/nix/store/6ffp6xbr6j2ryqw2dafbbvv55ggq16iv-bash-4.4-p23/bin/bash
Scanning dependencies of target alleg-dga2
Scanning dependencies of target allegro
[  1%] Building C object CMakeFiles/alleg-dga2.dir/src/x/xdga2.c.o
Scanning dependencies of target makedoc
[  1%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makedoc.c.o
In file included from /build/allegro-4.4.3.1/src/x/xdga2.c:26:
/nix/store/620j8vsvyq976lcxxszi99prcg4hh3s5-xorgproto-2019.1/include/X11/extensions/xf86dga.h:9:2: warning: #warning "xf86dga.h is obsolete and may be removed in the future." [-Wcpp]
    9 | #warning "xf86dga.h is obsolete and may be removed in the future."
      |  ^~~~~~~
/nix/store/620j8vsvyq976lcxxszi99prcg4hh3s5-xorgproto-2019.1/include/X11/extensions/xf86dga.h:10:2: warning: #warning "include <X11/extensions/Xxf86dga.h> instead." [-Wcpp]
   10 | #warning "include <X11/extensions/Xxf86dga.h> instead."
      |  ^~~~~~~
[  1%] Building C object CMakeFiles/allegro.dir/src/allegro.c.o
In file included from /nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/string.h:495,
                 from /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:35:
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:277:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 5 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:281:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 30 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:288:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 5 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:292:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 30 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:299:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 6 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_transform_custom_tags' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:303:3,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:331:7,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying 19 bytes from a string of the same length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In function 'strncpy',
    inlined from '_mangle_email_links' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:839:7,
    inlined from '_read_file' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:334:3,
    inlined from 'main' at /build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:185:8:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:106:10: warning: '__builtin_strncpy' output truncated before terminating nul copying as many bytes from a string as its length [-Wstringop-truncation]
  106 |   return __builtin___strncpy_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c: In function 'main':
/build/allegro-4.4.3.1/docs/src/makedoc/makedoc.c:839:7: note: length computed here
  839 |       strncpy(buf, temp, strlen(temp));
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[  1%] Building C object CMakeFiles/allegro.dir/src/blit.c.o
[  1%] Linking C shared module lib/alleg-dga2.so
[  2%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makehtml.c.o
[  2%] Built target alleg-dga2
Scanning dependencies of target copy_example_data
Scanning dependencies of target copy_shooter_data
[  2%] Generating allegro.pcx
[  2%] Generating demo.dat
[  2%] Built target copy_shooter_data
[  3%] Generating example.dat
Scanning dependencies of target copy_skater_data
[  4%] Generating demo.dat
/build/allegro-4.4.3.1/docs/src/makedoc/makehtml.c:74:18: warning: '_codeblock_end_len' defined but not used [-Wunused-const-variable=]
   74 | static const int _codeblock_end_len = 19;
      |                  ^~~~~~~~~~~~~~~~~~
[  4%] Generating examples.txt
[  4%] Generating exconfig.ini
[  4%] Generating level.dat
[  4%] Generating inkblot.tga
[  4%] Built target copy_skater_data
Scanning dependencies of target copy_setup_data
[  4%] Generating mysha.pcx
[  4%] Generating setup.dat
[  5%] Generating planet.pcx
[  5%] Built target copy_setup_data
[  5%] Generating running.dat
[  5%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makemisc.c.o
[  5%] Generating unifont.dat
[  5%] Built target copy_example_data
[  5%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/maketxt.c.o
[  5%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makertf.c.o
[  5%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/maketexi.c.o
[  6%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makeman.c.o
[  6%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makechm.c.o
[  6%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makedevh.c.o
[  6%] Building C object docs/CMakeFiles/makedoc.dir/src/makedoc/makesci.c.o
[  6%] Building C object CMakeFiles/allegro.dir/src/bmp.c.o
/build/allegro-4.4.3.1/docs/src/makedoc/makesci.c: In function '_convert':
/build/allegro-4.4.3.1/docs/src/makedoc/makesci.c:78:3: warning: ignoring return value of 'fread', declared with attribute warn_unused_result [-Wunused-result]
   78 |   fread(buf, sizeof(char), fs, tx_file); /* read in the file */
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[  7%] Building C object CMakeFiles/allegro.dir/src/clip3d.c.o
[  7%] Building C object CMakeFiles/allegro.dir/src/clip3df.c.o
[  7%] Linking C executable makedoc
[  7%] Built target makedoc
Scanning dependencies of target docs
[  8%] Generating txt/abi.txt
[  8%] Generating html/abi.html
[  8%] Generating txt/addons.txt
[  8%] Generating html/addons.html
[  8%] Generating txt/ahack.txt
[  8%] Building C object CMakeFiles/allegro.dir/src/colblend.c.o
[  9%] Generating html/ahack.html
[  9%] Building C object CMakeFiles/allegro.dir/src/color.c.o
[  9%] Generating txt/allegro.txt
[  9%] Generating html/allegro.html
[  9%] Generating txt/api.txt
[ 10%] Generating html/api.html
[ 10%] Generating txt/changes.txt
[ 10%] Generating html/changes.html
[ 10%] Generating txt/const.txt
[ 10%] Generating html/const.html
[ 11%] Generating txt/dat2c.txt
[ 11%] Generating html/dat2c.html
[ 11%] Generating txt/dat2s.txt
[ 11%] Generating html/dat2s.html
[ 11%] Generating txt/datafile.txt
[ 12%] Generating html/datafile.html
[ 12%] Generating txt/dat.txt
[ 12%] Generating html/dat.html
[ 12%] Generating txt/faq.txt
[ 12%] Generating html/faq.html
[ 13%] Generating txt/grabber.txt
[ 13%] Generating html/grabber.html
[ 13%] Generating txt/help.txt
[ 13%] Generating html/help.html
[ 13%] Generating txt/index.txt
[ 13%] Generating txt/license.txt
[ 14%] Generating html/index.html
[ 15%] Building C object CMakeFiles/allegro.dir/src/datafile.c.o
[ 15%] Generating html/license.html
[ 15%] Generating txt/makedoc.txt
[ 15%] Generating html/makedoc.html
[ 16%] Generating txt/mistakes.txt
[ 16%] Generating html/mistakes.html
/build/allegro-4.4.3.1/src/datafile.c: In function 'initialise_datafile':
/build/allegro-4.4.3.1/src/datafile.c:2151:12: warning: variable 's' set but not used [-Wunused-but-set-variable]
 2151 |    SAMPLE *s;
      |            ^
[ 16%] Generating txt/packfile.txt
[ 16%] Generating html/packfile.html
[ 16%] Generating txt/readme.txt
[ 17%] Generating html/readme.html
[ 17%] Generating txt/thanks.txt
[ 17%] Generating html/thanks.html
[ 17%] Generating build/cmake.txt
[ 17%] Generating build/cmake.html
[ 18%] Generating build/linux.txt
[ 18%] Generating build/linux.html
[ 18%] Generating build/macosx.txt
[ 18%] Generating build/macosx.html
[ 18%] Generating build/mingw32.txt
[ 19%] Generating build/mingw32.html
[ 19%] Generating build/msvc.txt
[ 19%] Generating build/msvc.html
[ 19%] Generating build/unix.txt
[ 19%] Generating build/unix.html
[ 20%] Generating AUTHORS
[ 20%] Generating THANKS
[ 20%] Generating CHANGES
[ 20%] Generating readme.txt
[ 20%] Generating man/allegro_init.3
[ 20%] Generating texi/allegro.texi
[ 20%] Generating scite/allegro.api
[ 21%] Generating info/allegro.info
[ 21%] Building C object CMakeFiles/allegro.dir/src/dataregi.c.o
[ 21%] Building C object CMakeFiles/allegro.dir/src/digmid.c.o
[ 21%] Building C object CMakeFiles/allegro.dir/src/dither.c.o
[ 21%] Building C object CMakeFiles/allegro.dir/src/dispsw.c.o
[ 22%] Building C object CMakeFiles/allegro.dir/src/drvlist.c.o
[ 22%] Building C object CMakeFiles/allegro.dir/src/file.c.o
utf8 "\xB0" does not map to Unicode at /nix/store/jjnl8b9gd6ri1bwfrkcbsz169qhj7asb-texinfo-6.7/share/texinfo/Texinfo/ParserNonXS.pm line 1796, <FH> line 18520.
Malformed UTF-8 character: \xb0 (unexpected continuation byte 0xb0, with no preceding start byte) in pattern match (m//) at /nix/store/jjnl8b9gd6ri1bwfrkcbsz169qhj7asb-texinfo-6.7/share/texinfo/Texinfo/ParserNonXS.pm line 3364.
Malformed UTF-8 character (fatal) at /nix/store/jjnl8b9gd6ri1bwfrkcbsz169qhj7asb-texinfo-6.7/share/texinfo/Texinfo/ParserNonXS.pm line 3364.
make[2]: *** [docs/CMakeFiles/docs.dir/build.make:435: docs/info/allegro.info] Error 25
make[1]: *** [CMakeFiles/Makefile2:647: docs/CMakeFiles/docs.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 22%] Building C object CMakeFiles/allegro.dir/src/fli.c.o
[ 22%] Building C object CMakeFiles/allegro.dir/src/font.c.o
[ 23%] Building C object CMakeFiles/allegro.dir/src/fontbios.c.o
[ 23%] Building C object CMakeFiles/allegro.dir/src/fontbmp.c.o
[ 23%] Building C object CMakeFiles/allegro.dir/src/fontdat.c.o
[ 23%] Building C object CMakeFiles/allegro.dir/src/fontgrx.c.o
[ 23%] Building C object CMakeFiles/allegro.dir/src/fonttxt.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/fsel.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/gfx.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/glyph.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/flood.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/graphics.c.o
[ 24%] Building C object CMakeFiles/allegro.dir/src/gsprite.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/gui.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/guiproc.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/inline.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/config.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/joystick.c.o
[ 25%] Building C object CMakeFiles/allegro.dir/src/keyboard.c.o
[ 26%] Building C object CMakeFiles/allegro.dir/src/lbm.c.o
[ 26%] Building C object CMakeFiles/allegro.dir/src/libc.c.o
[ 26%] Building C object CMakeFiles/allegro.dir/src/lzss.c.o
[ 26%] Building C object CMakeFiles/allegro.dir/src/math3d.c.o
[ 26%] Building C object CMakeFiles/allegro.dir/src/math.c.o
[ 27%] Building C object CMakeFiles/allegro.dir/src/midi.c.o
[ 27%] Building C object CMakeFiles/allegro.dir/src/mixer.c.o
[ 27%] Building C object CMakeFiles/allegro.dir/src/modesel.c.o
[ 27%] Building C object CMakeFiles/allegro.dir/src/mouse.c.o
[ 27%] Building C object CMakeFiles/allegro.dir/src/pcx.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/poly3d.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/polygon.c.o
/build/allegro-4.4.3.1/src/poly3d.c: In function 'draw_polygon_segment':
/build/allegro-4.4.3.1/src/poly3d.c:978:17: warning: left shift of negative value [-Wshift-negative-value]
  978 |      width = -1 << 16;
      |                 ^~
[ 28%] Building C object CMakeFiles/allegro.dir/src/quantize.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/quat.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/readbmp.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/rle.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/rotate.c.o
[ 28%] Building C object CMakeFiles/allegro.dir/src/rsfb.c.o
[ 29%] Building C object CMakeFiles/allegro.dir/src/scene3d.c.o
/build/allegro-4.4.3.1/src/scene3d.c: In function 'init_poly':
/build/allegro-4.4.3.1/src/scene3d.c:373:25: warning: this statement may fall through [-Wimplicit-fallthrough=]
  373 |             poly->flags |= INTERP_THRU;
/build/allegro-4.4.3.1/src/scene3d.c:374:10: note: here
  374 |          case DRAW_MODE_COPY_PATTERN:
      |          ^~~~
[ 29%] Building C object CMakeFiles/allegro.dir/src/sound.c.o
[ 29%] Building C object CMakeFiles/allegro.dir/src/spline.c.o
[ 29%] Building C object CMakeFiles/allegro.dir/src/stream.c.o
[ 29%] Building C object CMakeFiles/allegro.dir/src/text.c.o
[ 30%] Building C object CMakeFiles/allegro.dir/src/tga.c.o
[ 30%] Building C object CMakeFiles/allegro.dir/src/timer.c.o
[ 30%] Building C object CMakeFiles/allegro.dir/src/unicode.c.o
[ 30%] Building C object CMakeFiles/allegro.dir/src/vtable.c.o
[ 30%] Building C object CMakeFiles/allegro.dir/src/vtable15.c.o
[ 31%] Building C object CMakeFiles/allegro.dir/src/vtable16.c.o
[ 31%] Building C object CMakeFiles/allegro.dir/src/vtable24.c.o
[ 31%] Building C object CMakeFiles/allegro.dir/src/vtable32.c.o
[ 31%] Building C object CMakeFiles/allegro.dir/src/vtable8.c.o
[ 31%] Building C object CMakeFiles/allegro.dir/src/c/cblit16.c.o
[ 32%] Building C object CMakeFiles/allegro.dir/src/c/cblit24.c.o
[ 32%] Building C object CMakeFiles/allegro.dir/src/c/cblit32.c.o
[ 32%] Building C object CMakeFiles/allegro.dir/src/c/cblit8.c.o
[ 32%] Building C object CMakeFiles/allegro.dir/src/c/ccpu.c.o
[ 32%] Building C object CMakeFiles/allegro.dir/src/c/ccsprite.c.o
[ 33%] Building C object CMakeFiles/allegro.dir/src/c/cgfx15.c.o
[ 33%] Building C object CMakeFiles/allegro.dir/src/c/cgfx16.c.o
[ 33%] Building C object CMakeFiles/allegro.dir/src/c/cgfx24.c.o
[ 33%] Building C object CMakeFiles/allegro.dir/src/c/cgfx32.c.o
[ 34%] Building C object CMakeFiles/allegro.dir/src/c/cgfx8.c.o
[ 34%] Building C object CMakeFiles/allegro.dir/src/c/cmisc.c.o
[ 34%] Building C object CMakeFiles/allegro.dir/src/c/cscan15.c.o
[ 34%] Building C object CMakeFiles/allegro.dir/src/c/cscan16.c.o
[ 34%] Building C object CMakeFiles/allegro.dir/src/c/cscan24.c.o
[ 35%] Building C object CMakeFiles/allegro.dir/src/c/cscan32.c.o
[ 35%] Building C object CMakeFiles/allegro.dir/src/c/cscan8.c.o
[ 35%] Building C object CMakeFiles/allegro.dir/src/c/cspr15.c.o
[ 35%] Building C object CMakeFiles/allegro.dir/src/c/cspr16.c.o
[ 35%] Building C object CMakeFiles/allegro.dir/src/c/cspr24.c.o
[ 36%] Building C object CMakeFiles/allegro.dir/src/c/cspr32.c.o
[ 36%] Building C object CMakeFiles/allegro.dir/src/c/cspr8.c.o
[ 36%] Building C object CMakeFiles/allegro.dir/src/c/cstretch.c.o
[ 36%] Building C object CMakeFiles/allegro.dir/src/c/czscan15.c.o
[ 36%] Building C object CMakeFiles/allegro.dir/src/c/czscan16.c.o
[ 37%] Building C object CMakeFiles/allegro.dir/src/c/czscan24.c.o
[ 37%] Building C object CMakeFiles/allegro.dir/src/c/czscan32.c.o
[ 37%] Building C object CMakeFiles/allegro.dir/src/c/czscan8.c.o
[ 37%] Building C object CMakeFiles/allegro.dir/src/misc/ccolconv.c.o
[ 37%] Building C object CMakeFiles/allegro.dir/src/misc/colconv.c.o
[ 38%] Building C object CMakeFiles/allegro.dir/src/unix/alsa9.c.o
[ 38%] Building C object CMakeFiles/allegro.dir/src/unix/alsamidi.c.o
[ 38%] Building C object CMakeFiles/allegro.dir/src/unix/arts.c.o
[ 38%] Building C object CMakeFiles/allegro.dir/src/unix/sgial.c.o
[ 38%] Building C object CMakeFiles/allegro.dir/src/unix/jack.c.o
[ 39%] Building C object CMakeFiles/allegro.dir/src/unix/udjgpp.c.o
[ 39%] Building C object CMakeFiles/allegro.dir/src/unix/udrvlist.c.o
[ 39%] Building C object CMakeFiles/allegro.dir/src/unix/udummy.c.o
[ 39%] Building C object CMakeFiles/allegro.dir/src/unix/uesd.c.o
[ 39%] Building C object CMakeFiles/allegro.dir/src/unix/ufile.c.o
100 47.6M  100 47.6M    0     0  1311k      0  0:00:37  0:00:37 --:--:-- 2221k
building '/nix/store/q12g1vlz9al4a02lsl3mzqd89xnxbf6p-qtdeclarative-everywhere-src-5.15.0.tar.xz.drv'...
In file included from /nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/string.h:495,
                 from /build/allegro-4.4.3.1/src/unix/ufile.c:22:
In function 'strncat',
    inlined from 'al_findnext' at /build/allegro-4.4.3.1/src/unix/ufile.c:416:10:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:136:10: warning: '__builtin___strncat_chk' output truncated before terminating nul copying as many bytes from a string as its length [-Wstringop-truncation]
  136 |   return __builtin___strncat_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/src/unix/ufile.c: In function 'al_findnext':
/build/allegro-4.4.3.1/src/unix/ufile.c:34:28: note: length computed here
   34 |    #define NAMLEN(dirent) (strlen((dirent)->d_name))
      |                           ~^~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/src/unix/ufile.c:413:11: note: in expansion of macro 'NAMLEN'
  413 |       if (NAMLEN(entry) >= sizeof(tempname))
      |           ^~~~~~
In file included from /nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/string.h:495,
                 from /build/allegro-4.4.3.1/src/unix/ufile.c:22:
In function 'strncat',
    inlined from 'al_findnext' at /build/allegro-4.4.3.1/src/unix/ufile.c:421:10:
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/bits/string_fortified.h:136:10: warning: '__builtin___strncat_chk' output may be truncated copying between 0 and 1023 bytes from a string of length 1023 [-Wstringop-truncation]
  136 |   return __builtin___strncat_chk (__dest, __src, __len, __bos (__dest));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 40%] Building C object CMakeFiles/allegro.dir/src/unix/ugfxdrv.c.o
[ 40%] Building C object CMakeFiles/allegro.dir/src/unix/ujoydrv.c.o
[ 40%] Building C object CMakeFiles/allegro.dir/src/unix/ukeybd.c.o
[ 40%] Building C object CMakeFiles/allegro.dir/src/unix/umain.c.o
[ 40%] Building C object CMakeFiles/allegro.dir/src/unix/umodules.c.o

trying https://download.qt.io/official_releases/qt/5.15/5.15.0/submodules/qtdeclarative-everywhere-src-5.15.0.tar.xz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
[ 41%] Building C object CMakeFiles/allegro.dir/src/unix/umouse.c.o
[ 41%] Building C object CMakeFiles/allegro.dir/src/unix/uoss.c.o
/build/allegro-4.4.3.1/src/unix/uoss.c: In function 'oss_update':
/build/allegro-4.4.3.1/src/unix/uoss.c:177:3: warning: ignoring return value of 'write', declared with attribute warn_unused_result [-Wunused-result]
  177 |   write(oss_fd, oss_bufdata, oss_bufsize);
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 41%] Building C object CMakeFiles/allegro.dir/src/unix/uossmidi.c.o
100   304  100   304    0     0    638      0 --:--:-- --:--:-- --:--:--   638
/build/allegro-4.4.3.1/src/unix/uossmidi.c: In function 'seq_set_fm_patches':
/build/allegro-4.4.3.1/src/unix/uossmidi.c:264:7: warning: ignoring return value of 'write', declared with attribute warn_unused_result [-Wunused-result]
  264 |       write(fd, &ins, sizeof(ins));
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/src/unix/uossmidi.c:272:7: warning: ignoring return value of 'write', declared with attribute warn_unused_result [-Wunused-result]
  272 |       write(fd, &ins, sizeof(ins));
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
/nix/store/bgvxpx97q789xc8rrginprlvk8p03p93-glibc-2.31-dev/include/linux/soundcard.h: In function 'seqbuf_dump':
/build/allegro-4.4.3.1/src/unix/uossmidi.c:110:7: warning: ignoring return value of 'write', declared with attribute warn_unused_result [-Wunused-result]
  110 |       write(seq_fd, _seqbuf, _seqbufptr);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 41%] Building C object CMakeFiles/allegro.dir/src/unix/uptimer.c.o
[ 41%] Building C object CMakeFiles/allegro.dir/src/unix/usigalrm.c.o
[ 42%] Building C object CMakeFiles/allegro.dir/src/unix/usnddrv.c.o
[ 42%] Building C object CMakeFiles/allegro.dir/src/unix/ustimer.c.o
[ 42%] Building C object CMakeFiles/allegro.dir/src/unix/usystem.c.o
/build/allegro-4.4.3.1/src/unix/usystem.c: In function '_unix_get_executable_name':
/build/allegro-4.4.3.1/src/unix/usystem.c:412:7: warning: ignoring return value of 'fgets', declared with attribute warn_unused_result [-Wunused-result]
  412 |       fgets(linkname, sizeof(linkname), pipe);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/src/unix/usystem.c:420:7: warning: ignoring return value of 'fgets', declared with attribute warn_unused_result [-Wunused-result]
  420 |       fgets(linkname, sizeof(linkname), pipe);
      |       ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/build/allegro-4.4.3.1/src/unix/usystem.c: In function '_find_executable_file':
/build/allegro-4.4.3.1/src/unix/usystem.c:276:3: warning: ignoring return value of 'getcwd', declared with attribute warn_unused_result [-Wunused-result]
  276 |   getcwd(pathname, sizeof(pathname));
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 42%] Building C object CMakeFiles/allegro.dir/src/unix/uthreads.c.o
[ 42%] Building C object CMakeFiles/allegro.dir/src/unix/utimer.c.o
[ 43%] Building C object CMakeFiles/allegro.dir/src/linux/ljoy.c.o
[ 43%] Building C object CMakeFiles/allegro.dir/src/x/xgfxdrv.c.o
[ 43%] Building C object CMakeFiles/allegro.dir/src/x/xkeyboard.c.o
/build/allegro-4.4.3.1/src/x/xkeyboard.c: In function '_xwin_get_keyboard_mapping':
/build/allegro-4.4.3.1/src/x/xkeyboard.c:620:3: warning: 'XKeycodeToKeysym' is deprecated [-Wdeprecated-declarations]
  620 |   KeySym sym = XKeycodeToKeysym(_xwin.display,
      |   ^~~~~~
In file included from /build/allegro-4.4.3.1/src/x/xkeyboard.c:24:
/nix/store/7mcscy0in56fl6wlz9rkzlz470f5p43g-libX11-1.6.8-dev/include/X11/Xlib.h:1687:15: note: declared here
 1687 | extern KeySym XKeycodeToKeysym(
      |               ^~~~~~~~~~~~~~~~
[ 43%] Building C object CMakeFiles/allegro.dir/src/x/xmouse.c.o
[ 43%] Building C object CMakeFiles/allegro.dir/src/x/xsystem.c.o
[ 44%] Building C object CMakeFiles/allegro.dir/src/x/xtimer.c.o
[ 44%] Building C object CMakeFiles/allegro.dir/src/x/xvtable.c.o
[ 44%] Building C object CMakeFiles/allegro.dir/src/x/xwin.c.o
[ 44%] Building C object CMakeFiles/allegro.dir/src/x/xdga2.c.o
[ 44%] Building C object CMakeFiles/allegro.dir/src/readsmp.c.o
[ 45%] Building C object CMakeFiles/allegro.dir/src/readfont.c.o
[ 45%] Linking C shared library lib/liballeg.so
[ 45%] Built target allegro
make: *** [Makefile:149: all] Error 2
builder for '/nix/store/hf7q7rq1ndrfd9nlbg1qyidf4xr4jxhn-allegro-4.4.3.1.drv' failed with exit code 2
cannot build derivation '/nix/store/h1gw5j09cwshn1lp3387ycaqhyw4lr16-lutris-usr-target.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/cjy3h02c4mf7ah7mbff9d2vakikanj9n-lutris-fhs.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/1bazi1kvs7n7rag87gf5m4aibanlz4fw-lutris.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/dqc6vkzsh0f87zkkx760cwha72yqnmpb-lutris.drv': 1 dependencies couldn't be built
error: build of '/nix/store/dqc6vkzsh0f87zkkx760cwha72yqnmpb-lutris.drv' failed
```

</details>

**Expected behavior**

**Screenshots**

**Additional context**

**Notify maintainers**
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@chiiruno

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.7.10-zen1, NixOS, 20.09pre-git (Nightingale)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.7`
 - channels(root): `"nixos-20.09pre237686.96069f7d890"`
 - nixpkgs: `/run/current-system/nixpkgs`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/development/libraries/allegro/default.nix
+++ b/pkgs/development/libraries/allegro/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchurl, texinfo, libXext, xorgproto, libX11
+{ stdenv, fetchurl, texinfo6_5, libXext, xorgproto, libX11
 , libXpm, libXt, libXcursor, alsaLib, cmake, zlib, libpng, libvorbis
 , libXxf86dga, libXxf86misc
 , libXxf86vm, openal, libGLU, libGL }:
@@ -17,7 +17,7 @@ stdenv.mkDerivation rec {
   ];
 
   buildInputs = [
-    texinfo libXext xorgproto libX11 libXpm libXt libXcursor
+    texinfo6_5 libXext xorgproto libX11 libXpm libXt libXcursor
     alsaLib cmake zlib libpng libvorbis libXxf86dga libXxf86misc
     libXxf86vm openal libGLU libGL
   ];

